Framework/Algorithms/src/PDCalibration.cpp

```diff
@@ -590,8 +607,8 @@ void PDCalibration::exec() {
      // height of fitted peak centers, default `nan` for failed fitted peaks
      std::vector<double> height_vec_full(numPeaks, std::nan(""));
      std::vector<double> weights; // weights for diff const fits
-     // for (size_t i = 0; i < fittedTable->rowCount(); ++i) {
-     const size_t rowNumInFitTableOffset = wkspIndex * numPeaks;
+     // row where first peak occurs
+     const size_t rowNumInFitTableOffset = (wkspIndex - m_startWorkspaceIndex) * numPeaks;
```

Comment:
`const size_t rowNumInFitTableOffset = (wkspIndex - m_startWorkspaceIndex + 1) * numPeaks;`?

Reply:
The rows start counting from zero.